x-pack/test/security_api_integration/tests/login_selector/basic_functionality.ts

```diff
@@ -97,11 +97,23 @@ export default function ({ getService }: FtrProviderContext) {
       // to fully authenticate user yet.
       const intermediateAuthCookie = request.cookie(handshakeResponse.headers['set-cookie'][0])!;
 
+      // When login page is accessed directly.
       await supertest
         .get('/login')
         .ca(CA_CERT)
         .set('Cookie', intermediateAuthCookie.cookieString())
         .expect(200);
+
+      // When user tries to access any other page in Kibana.
```

Comment:
__note:__ that is the main test in this PR, I made sure it fails without my change.